developer/frameworks/articles/290-configuration/01-categorizing-configurations.markdown

```diff
@@ -0,0 +1,108 @@
+# Categorizing the Configuration [](id=categorizing-the-configuration)
+
+By default, the configuration UI for your app is generated in *System Settings*
+&rarr; *Platform* &rarr; *Third Party*. You probably don't really want it
+there; by categorizing your configuration you can place it somewhere intuitive
+and easy to find.
+
+| Note: If you
+| [scope](/docs/7-2/frameworks/-/knowledge_base/f/scoping-configurations) your
+| configuration so that a UI is generated in Instance Settings as well, your
+| categorization will apply to that UI also.
+
+You have two options: either locate your configuration UI in an existing
+category and section, or create your own.
+
+Here are the default  System Settings sections. All available categories are
+nested beneath these sections:
+
+1.  Content Management
+
+2.  Social
+
+3.  Platform
+
+4.  Security 
+
+5.  Commerce
+
+6.  Other
```

Comment:
"Content" and "Social" were replaced by "Content and Data" in issue https://issues.liferay.com/browse/LPS-94972